packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -162,10 +167,16 @@ export function ArticleEditor({id}: ArticleEditorProps) {
         socialMediaAuthors,
         socialMediaImage
       } = latest
-      const {publishedAt} = published ?? {}
 
+      const {publishedAt} = latest ?? {}
       if (publishedAt) setPublishedAt(new Date(publishedAt))
 
+      const {updatedAt} = latest ?? {}
+      if (updatedAt) setUpdatedAt(new Date(updatedAt))
+
+      const {publishAt} = pending ?? {}
```

Comment:
I think here it should be `pending ?? published`. Once an article is published it still has an `publishAt` date.